app/adapters/external/cache/infinite_buying_adapter.py

```
@@ -523,16 +670,47 @@ async def restore_state(self, market: str, backup_data: dict[str, Any]) -> bool:
+                if len(parts) >= REDIS_CONFIG_KEY_PARTS_COUNT:
+                    market = parts[1]
+
+                    # 해당 마켓의 상태가 활성인지 확인
+                    state = await self.get_state(market)
+                    if state and state.is_active:
+                        active_markets.append(market)
+
+            logger.debug(
+                f"활성 마켓 조회 완료 - count: {len(active_markets)}, markets: {active_markets}"
+            )
+            return active_markets
```

Ruff (0.11.9)

Consider moving this statement to an `else` block

(TRY300)